### PR TITLE
Gives Pruno a new sprite, along with several minor changes to the brewing process codewise

### PR DESCRIPTION
**Pull request by minichibis**
## About The Pull Request

Pruno now has a unique sprite when in the glass, a frothing, bubbling mixture coated with mold, served in a cracked wine glass. Drink up!
pruno
In addition, Pruno Mix has a sprite if you decide to pour it into a glass on its own for some reason.
image
Finally, pruno has had some minor changes across the board:
-the bag of pruno mix used to magically produce pruno even when it had no mix. The process has been changed to only trigger when pruno mixture is actually in the bag.
-a new reagent has been added, fermented pruno mix. This slurry is to make sure that brewing does not happen multiple times concurrently.
-disgust values for pruno have been lowered from 5 to 1, as well as adding disgust values to pruno mixture and its fermented variant. This should allow prisoners to get drunk and puke out their guts, rather than just puking out their guts.
image

## Why It's Good For The Game

A generic sprite for such a unique drink is a waste in my eyes, and some new graphics should really help the drink feel a bit more flavorful.
The decreased disgust will also make it serve its purpose as alcohol better, allowing inmates to get hammered and slur their speech before they start to vomit.

## Changelog
:cl: minichibis

imageadd: Prisons galaxy-wide have been shipped a large quantity of chipped wine glasses and plastic baggies due to a bureaucratic error. Security should keep a close eye on reports of unlicensed alcohol in permabrig.
balance: Pruno now is a bit easier to swallow without throwing up.
fix: Prisoners can no longer brew infinite supplies of alcohol out of the same empty bag of pruno mix.

/:cl:

### CODE REVIEW

**Review by Twaticus — CHANGES_REQUESTED** [review]
For the glass of pruno, the animation looks great awesome job. It could do with a bit of shine on the edge to match the other glass drinks and the crack could be darkened with a touch of highlight to make it more pronounced.

For the pruno mix glass/fermented pruno mix glass, they could use some darker edges so they dont blend into the backround as easily, and the bags themselves need some sort of transparent back fill. Other than that, the fermented version really does give that moldy vibe which is great :+1:

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -568,6 +568,9 @@
 // TODO: make it so the washer spills reagents if a reagent container is in there, for now, you can wash pruno
 
 /obj/item/reagent_containers/food/drinks/bottle/pruno/proc/check_fermentation()
+	if ((!reagents) || (!reagents.has_reagent(/datum/reagent/consumable/prunomix)))
```
Comment (Rohesie) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm:571]:
```suggestion
	if (!reagents?.has_reagent(/datum/reagent/consumable/prunomix))
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -912,10 +912,27 @@
...
 	description = "Fruit, sugar, yeast, and water pulped together into a pungent slurry."
 	taste_description = "garbage"
-	glass_icon_state = "glass_orange"
+	glass_icon_state = "pruno_mix"
 	glass_name = "glass of pruno mixture"
 	glass_desc = "Fruit, sugar, yeast, and water pulped together into a pungent slurry."
 
+/datum/reagent/consumable/prunomix/on_mob_life(mob/living/carbon/M)
+	M.adjust_disgust(0.5)
+	..()
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:921]:
```suggestion
	return ..()
```

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -912,10 +912,27 @@
...
+	color = "#E78108"
+	description = "The disgusting byproduct of making alcohol in the most desperate way possible. Often drunk alongside the actual drink straight from the bag."
+	taste_description = "a mass grave for fruit"
+	glass_icon_state = "pruno_mix_fermented"
+	glass_name = "glass of fermented pruno mixture"
+	glass_desc = "The disgusting byproduct of making alcohol in the most desperate way possible. Often drunk alongside the actual drink straight from the bag."
+
+/datum/reagent/consumable/prunomixfermented/on_mob_life(mob/living/carbon/M)
+	M.adjust_disgust(1)
+	..()
```
Comment (Rohesie) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:934]:
```suggestion
	return ..()
```

**Comment by ExcessiveUseOfCobblestone** [comment]
RePR once review and MC are completed.

**Pull request closed without merging**